R/entry_functions.R

```diff
@@ -97,3 +97,30 @@ get_entry_picks <- function(entryid = NULL, gw = NULL){
   return(picks2)
 }
 
+get_entry_captain <- function(entryid = NULL, gw = NULL, player_names = FALSE){
+  if(is.null(entryid)) stop("You'll need to input an entry ID, mate.")
+  if(is.null(gw)) stop("You'll need to input a gameweek, mate.")
+  
+  captain3 <- data.frame()
+  for (i in 1:length(gw)) {
+    captain2 <- data.frame()
+    for (j in 1:length(entryid)) {
+      picks <- jsonlite::fromJSON(paste("https://fantasy.premierleague.com/api/entry/",entryid[j],"/event/",gw[i],"/picks","/",sep=""))
+      captain <- data.frame("id" = picks$picks[picks$picks$multiplier %in% c(2, 3), "element"])
+      captain$entry <- entryid[j]
+      captain2 <- rbind(captain2, captain)
+    }
+    captain2$event <- gw[i]
+    captain3 <- rbind(captain3, captain2)
+  }
+  
+  if (player_names == TRUE) {
+    player_names <- get_player_name(unique(captain3$captain))
+    captain3 <- merge(captain3, player_names, by = "id")
+    captain3 <- captain3[c(1, 4, 2, 3)]
+    names(captain3) <- c("captain_id", "captain_name", "entry", "event")
+  }
```

Comment:
Cf. above, remove the if() statement wrapper here.

Would also add a reordering here to sort the return by entry and event
```suggestion
captain3 <- captain3[order(captain3$entry,captain3$event),]
```